<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="HNS_Time_Button" name="PB_Time">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Mono</family>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="HNS_VKP" name="GV_VKP" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>234</width>
      <height>348</height>
     </rect>
    </property>
   </widget>
   <widget class="HNS_Date_Button" name="PB_Date">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Mono</family>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Test">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Sort">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Test</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNS_VKP</class>
   <extends>QWidget</extends>
   <header>hns_vkp.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HNS_Time_Button</class>
   <extends>QPushButton</extends>
   <header>hns_time_button.h</header>
  </customwidget>
  <customwidget>
   <class>HNS_Date_Button</class>
   <extends>QPushButton</extends>
   <header>hns_date_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
